logistics/scripts/modules/LoadTools.psm1

```diff
@@ -11,46 +11,48 @@ Import-Module -Force -Scope Global (Get-RepositoryResolvedPath "logistics\script
 Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'logistics\scripts\modules\packaging\restore-packages.psm1')
 Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'logistics\scripts\modules\settings\settings-management.psm1')
 
-function Invoke-RestoreLoadToolsPackages {
-    param(
-        [hashtable] $config
-    )
-
-    $toolsPath = Join-Path (Get-RepositoryResolvedPath) 'tools'
-    Restore-Packages -toolsPath $toolsPath -solutionPath (Get-ChildItem $config.loadToolsSolution).FullName
-}
-
 function Invoke-BuildLoadTools {
     param(
         [hashtable] $config
     )
 
     $buildConfiguration = $config.buildConfiguration
+    $solutionPath = $config.loadToolsSolution
+    $verbosity = "minimal"
+
+    if ((Get-DeploymentSettings).Engine -eq 'PostgreSQL') { $buildConfiguration = 'Npgsql' }
```

Comment:
This line seems to be causing the template builds to fail. We don't have a `Npgsql` build configuration anymore so we should be able to remove it.
```suggestion
```